src/popup.html

```diff
@@ -1,7 +1,7 @@
 <html data-theme="auto">
 <head>
   <meta http-equiv="content-type" content="text/html; charset=utf-8">
-  <title>Multi-Account Containers</title>
+  <title>Firefox Multi-Account Containers</title>
```

Comment:
question (non-blocking): Should these hard-coded English titles be replaced with translations? I don't see an attribute identifier for JS to do any replacement here.

Reply:
AFAIK, we don't translate the product name anywhere (e.g. SUMO KBs) and it's the same for other Mozilla products so that's why I didn't feel like we needed an i18n string here. However, if you think it'd be a good idea, I don't have any strong opinion about it.